platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init

```diff
@@ -38,6 +38,10 @@ start)
         modprobe dx010_wdt
         modprobe leds-dx010
         modprobe lm75
+        sleep 1
+        modprobe -r gpio_ich
```

Comment:
use blacklist feature.

Reply:
Add the blacklist in #2691

Reply:
The module unload lines will be removed after #2691 merged.